src/pages/Risk.tsx

```diff
@@ -12,15 +12,15 @@ function Risk(): ReactElement {
     <div className="riskpage">
       <TopMenu activeTab={t("risk")} />
       <div className="content">
-        <p>
+        <p data-cy="risk-intro">
```

Comment:
using cypress recommended convention for targeting

Reply:
we also use `data-testid` in a few places. Since this is just a data attribute it shouldn't matter which we choose, but I think I'd prefer to just pick one or the other to be consistent.

Reply:
i can change it to `data-testid` as it's only important that it's unique and use that going forward.